manager/assets/modext/util/datetime.js

```diff
@@ -30,6 +30,11 @@ Ext.ux.form.DateTime = Ext.extend(Ext.form.Field, {
      * and submitted to server (defaults to 'Y-m-d H:i:s' that is mysql format)
      */
     ,hiddenFormat:'Y-m-d H:i:s'
+    /**
+     * @cfg {String} hiddenFormat Format of datetime used to store value in hidden field
+     * and submitted to server (defaults to 'Y-m-d H:i:s' that is mysql format)
```

Comment:
The description has to be slightly changed, but otherwise ok.

Reply:
Thanks @Jako, you have to re-approve this PR.